app/controllers/comments_controller.rb

```diff
@@ -14,5 +15,10 @@ def destroy
 
 		redirect_to post_path(@post)
 	end
+
+	private
+		def is_current_user_owner(post)
```

Comment:
looks like you are not even using this method?

Reply:
I added this for future use, I'll remove it if I don't find a purpose later.